target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,13 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_usbdev "phone" "Phone" "${board}:white:phone" "1"
+	ucidef_set_led_usbdev "usb" "USB" "${board}:white:usb" "1"
+	ucidef_set_led_netdev "lan" "LAN" "${board}:white:lan" "br-lan"
+	ucidef_set_led_netdev "dsl" "DSL" "${board}:white:dsl" "1"
+	ucidef_set_led_default "general" "general" "${board}:white:ledgnr" "1"
```

Comment:
Why do you power on this led here? This LED is already handled by diag.sh due to the led-failsafe = &ledgnr in your dts.